module_1/first-react-app/src/css/Movie.css

```diff
@@ -0,0 +1,30 @@
+/* CSS */
+.movie-tile {
+    min-height: 75%;
+    height: 100%;
+    object-fit: cover;
+    border: 1px solid black;
+    box-shadow: 0 6px 12px rgba(0, 100, 0, 2);
+}
+
+.title {
+    color: white;
+    font-size: 1.1em;
+    max-width: 70%;
+}
+
+.genre {
+    padding-top: 10px;
+    color: grey;
+    font-size: 1em;
+    overflow: hidden;
+}
+
+.movie-description {
+    height: 25% !important;
```

Comment:
Can we avoid `!important`?

Reply:
We have used this, as the style was being overridden by materialize css